docs/self-managed/console-deployment/configuration.md

```diff
@@ -17,21 +17,26 @@ Underscores in environment variables correspond to configuration file key levels
+| `CAMUNDA_IDENTITY_CLIENT_ID`     | Client Id for Console client                                                                                                                                                                                                                                                       | console                                  |
+| `CAMUNDA_CONSOLE_CONTEXT_PATH`   | Context path for Console                                                                                                                                                                                                                                                           | console                                  |
+| `CAMUNDA_CONSOLE_CUSTOMERID`     | Unique identifier of the customer                                                                                                                                                                                                                                                  | `customer-id`                            |
+| `CAMUNDA_CONSOLE_INSTALLATIONID` | Unique installation id of the current customer installation                                                                                                                                                                                                                        | `installation-id`                        |
+| `CAMUNDA_CONSOLE_TELEMETRY`      | Telemetry config for Console Self-Managed: `disabled`, `online`, or `download`                                                                                                                                                                                                     | `online`                                 |
+| `CAMUNDA_CONSOLE_DISABLE_AUTH`   | Disables authentication for Console. With this option, set users don't have to log in to use Console and API requests can be executed without an Authorization header. <br /> By disabling authentication all `CAMUNDA_IDENTITY`, variables won't be used.                         | `true`                                   |
+| `CAMUNDA_LICENSE_KEY`            | Your Camunda 8 license key, if your installation requires a license. For Helm installations, license keys can be configured globally in your `values.yaml` file. See the [Helm installation documentation](/self-managed/setup/install.md#configure-license-key) for more details. | N/A                                      |
 
 Console environment variables could be set in Helm via the `console.env` key. For more details, check [Console Helm values](https://artifacthub.io/packages/helm/camunda/camunda-platform#console-parameters).
 
+:::note
+Camunda 8 components without a valid license may display **Non-Production License** in the navigation bar and issue warnings in the logs. These warnings have no impact on startup.
```

Comment:
I think we can call that key will not affect functionality of Console
'have no impact on startup or functionality.'

Reply:
Adjusted in latest commit 👍